http/src/main/resources/reference.conf

```diff
@@ -377,9 +377,9 @@ proxy {
   proxyPort = 443
   dnsPollPeriod = 15 seconds
   tokenCacheExpiryTime = 60 minutes
-  tokenCacheMaxSize = 10000
+  tokenCacheMaxSize = 20000
```

Comment:
Just a guess really.. the `CacheMetrics` in this PR should help inform us more about tuning these caches.

Was also thinking about analyzing a production heap dump for potential improvements there.